diff --git a/node_modules/youtube-mp3-downloader/typings/index.d.ts b/node_modules/youtube-mp3-downloader/typings/index.d.ts
index 255bcc1..895fa96 100644
--- a/node_modules/youtube-mp3-downloader/typings/index.d.ts
+++ b/node_modules/youtube-mp3-downloader/typings/index.d.ts
@@ -35,7 +35,7 @@ declare class YoutubeMp3Downloader {
 
   cleanFileName(fileName: string): string;
   download(videoId: string, fileName?: string): void;
-  performDownload(task, callback: (errorNessage?: string, output?: any) => void): void;
+  performDownload(task: any, callback: (errorNessage?: string, output?: any) => void): void;
 
   on(event: 'queueSize', listener: (total : number) => void): this;
   on(event: 'error' | 'finished', listener: (err: any, data: any) => void): this;
